app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+from typing import Any, Callable
+from functools import wraps
+
+
+class CommandError(Exception):
+    """Checks command input for right format."""
+
+
+def check_command(func: Callable) -> Callable:
+    @wraps(func)
+    def wrapper(command: str) -> Any:
+        if command.split()[0] != "mv" or len(command.split()) != 3:
+            raise CommandError("Please enter valid command!")
+        return func(command)
+
+    return wrapper
+
+
+@check_command
+def move_file(command: str) -> None:
+    _, source_name, target_name = command.split()
+    destination_path = os.path.join(target_name)
```

Comment:
why do you need this? you can make it without the need to assign it to destination_path separately